spec/features/accessibility_spec.rb

```diff
@@ -0,0 +1,180 @@
+        path.gsub!("/start", "/fix-choice") if path.include?("/bulk-upload-resume/#{bulk_upload.id}/start")
+        visit(path)
+        expect(page).to have_current_path(path)
+        expect(page).to be_axe_clean.according_to :wcag2aa
+      end
+    end
+  end
+
+  context "when viewing sales log pages" do
+    let(:bulk_upload) { create(:bulk_upload, user:) }
+    let(:sales_log) { create(:sales_log, :completed, assigned_to: other_user, bulk_upload_id: bulk_upload.id) }
+    let(:organisation_relationship) { create(:organisation_relationship, parent_organisation: user.organisation) }
+
+    let(:sales_log_paths) do
+      all_page_ids = FormHandler.instance.sales_forms.values.flat_map(&:pages).map(&:id).uniq
+      sales_log_pages = sales_log.form.pages
+      other_form_page_ids = all_page_ids - sales_log_pages.map(&:id)
+
+      routes = find_routes("sales-log", sales_log, bulk_upload)
+
+      routes.reject { |path|
+        path.include?("/edit") || path.include?("/new") || path.include?("*page") ||
+          path.include?("/sales-logs/bulk-upload-logs/#{bulk_upload.id}") ||
+          path.include?("bulk-upload-soft-validations-check") || path.include?("filters/update") ||
+          path == "/sales-logs/bulk-upload-resume/#{bulk_upload.id}" ||
+          path == "/sales-logs/bulk-upload-logs" ||
+          other_form_page_ids.any? { |page_id| path.include?(page_id.dasherize) } ||
+          sales_log_pages.any? { |page| path.include?(page.id.dasherize) && !page.routed_to?(sales_log, user) }
+      }.uniq
+    end
+
+    it "is has accessible pages" do
+      sales_log_paths.each do |path|
+        path += "?original_log_id=#{sales_log.id}" if path.include?("duplicate")
+        path += "?codes_only=true&years[]=2024" if path.include?("csv")
+        path.gsub!("/start", "/prepare-your-file?form[year]=2024") if path.include?("bulk-upload-logs/start")
+        path.gsub!("/start", "/fix-choice") if path.include?("/bulk-upload-resume/#{bulk_upload.id}/start")
+
+        visit(path)
+        expect(page).to have_current_path(path)
+        expect(page).to be_axe_clean.according_to :wcag2aa
+      end
+    end
+  end
+
+  context "when viewing scheme pages" do
+    let(:scheme) { create(:scheme, owning_organisation: other_user.organisation) }
+    let!(:location) { create(:location, scheme:) }
+    let(:scheme_paths) do
+      routes = find_routes("scheme", scheme, location)
+
+      routes.reject { |path|
+        path.include?("/edit") || path.include?("/new") || path.include?("*page") ||
+          path.include?("reactivate") || path.include?("deactivate")
+      }.uniq
+    end
+
+    before do
+      allow(FormHandler.instance).to receive(:in_crossover_period?).and_return(true)
+    end
+
+    it "is has accessible pages" do
+      scheme_paths.each do |path|
+        visit(path)
+        expect(page).to have_current_path(path)
+        expect(page).to be_axe_clean.according_to :wcag2aa
+      end
+    end
+  end
+end
```

Comment:
Do we need to check the homepage / guidance page?

Reply:
Yup! Added some